### PR TITLE
Pass DracoLoader to Tile3DLayer from app

### PR DESCRIPTION
**Pull request by xintongxia**
#### Change List
- depend on #412

### CODE REVIEW

File: modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js [hunk 1]
```diff
@@ -156,10 +159,6 @@ export default class Tile3DLayer extends CompositeLayer {
     this._selectLayers(frameNumber);
   }
 
-  _getDracoLoader() {
-    return this.props.DracoWorkerLoader || this.props.DracoLoader;
```
Comment (Pessimistress) [inline modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js]: Do we still need these props?

Comment (xintongxia) [reply modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js]: updated doc

File: docs/layers/tile-3d-layer.md [hunk 2]
```diff
@@ -121,15 +121,6 @@ const layer = new Tile3DLayer({
 })
 ```
 
-##### `DracoLoader` (Object, Optional)
```
Comment (ibgreen) [inline docs/layers/tile-3d-layer.md]: We should cut this, but we must instead add very clear instructions to the user about how the DracoLoader should be installed.

File: examples/website/3d-tiles/app.js [hunk 3]
```diff
@@ -4,9 +4,11 @@ import {StaticMap} from 'react-map-gl';
 import DeckGL from '@deck.gl/react';
 import {Tile3DLayer} from '@deck.gl/geo-layers';
 
+import {registerLoaders} from '@loaders.gl/core';
 // To manage dependencies and bundle size, the app must decide which supporting loaders to bring in
 import {DracoLoader, DracoWorkerLoader} from '@loaders.gl/draco';
 
+registerLoaders([DracoWorkerLoader, DracoLoader]);
```
Comment (Pessimistress) [inline examples/website/3d-tiles/app.js]: Do we need both here? This adds twice the draco decoder to the website bundle.

**Pull request merged**